### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by AntoineAugusti**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,15 +33,14 @@
         'Jinja2==2.11.3',
         'statsd==3.3.0',
         'Flask-Redis==0.4.0',
-        'PyYAML==5.3.1',
-        'phonenumbers==8.12.12',
+        'PyYAML==5.4.1',
+        'phonenumbers==8.12.21',
         'pytz==2021.1',
```
Comment (AntoineAugusti) [inline setup.py:38]: Removed monotonic because it's available since Python 3.3 and we run Python 3.9

File: requirements_for_test.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+pytest==6.2.3
+pytest-mock==3.5.1
+pytest-cov==2.11.1
+pytest-xdist==2.2.1
 requests-mock==1.8.0
-freezegun==0.3.15
-flake8==3.8.4
-flake8-print==3.1.4
+freezegun==1.1.0
+flake8==3.9.1
```
Comment (QuanNguyen-CDS) [inline requirements_for_test.txt:8]: are there any conflicts with notification-utils and some of the other projects that depend upon this project? I'm seeing some differences between this and notification-api and notification-admin, i.e flake=3.8.4 within -api and -admin but is getting updated within -utils

File: setup.py [hunk 3]
```diff
@@ -33,15 +33,14 @@
...
         'smartypants==2.0.1',
-        'monotonic==1.5',
         'pypdf2==1.26.0',
 
         # required by both api and admin
-        'awscli==1.19.0',
-        'boto3==1.17.0',
+        'awscli==1.19.57',
+        'boto3==1.17.57',
```
Comment (QuanNguyen-CDS) [inline setup.py:44]: should we update to the latest? boto3 is at 1.17.58, awscli is at 1.19.58 (both released on April 26 so probably released on the same day that you posted this PR)

File: requirements_for_test.txt (hunk 2 above)
Comment (AntoineAugusti) [reply requirements_for_test.txt:8]: The plan is to bump utils first, pull the new version on admin and API and update dependencies there as well

File: setup.py (hunk 3 above)
Comment (AntoineAugusti) [reply setup.py:44]: Will do

**Review by QuanNguyen-CDS — APPROVED** [review]

**Pull request merged**